addons/account/security/ir.model.access.csv

```diff
@@ -84,3 +84,4 @@ access_account_cashbox_line,account.bank.statement.cashbox.line,model_account_ca
 access_account_tax_group_internal_user,account.tax.group internal user,model_account_tax_group,base.group_user,1,0,0,0
 access_account_tax_group,account.tax.group,model_account_tax_group,account.group_account_invoice,1,0,0,0
 access_account_tax_group_manager,account.tax.group,model_account_tax_group,account.group_account_manager,1,1,1,1
+access_account_fiscalyear_group_manager,account.fiscalyear.group.manager,model_account_fiscalyear,account.group_account_manager,1,1,1,1
```

Comment:
you will need read access for the rest of the other users **employees for example**.

and several checks will depend of it, then review do not left portal users tests for the end.

Reply:
he's right: we need at least to give read right to regular account users